src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    this.hidden = false;
```

Comment:
as task description says:

> Create a Herbivore class. Herbivore has a method of `hide`

It means that only herbivores can hide (mot every Animal). In your case every instance will have `hidden` property, which should be onli in herbivores